sycl/include/sycl/exception.hpp

```diff
@@ -16,6 +16,7 @@
 #include <sycl/detail/defines_elementary.hpp> // for __SYCL2020_DEPRECATED
 #include <sycl/detail/export.hpp>             // for __SYCL_EXPORT
 #include <sycl/detail/pi.h>                   // for pi_int32
+#include <sycl/detail/string.hpp>
```

Comment:
Should we include under __INTEL_PREVIEW_BREAKING_CHANGES?

Reply:
Sure. Done.